cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -58,6 +58,9 @@ $GCE_METADATA_SERVER = "169.254.169.254"
 # exist until an initial HNS network has been created on the Windows node - see
 # Add_InitialHnsNetwork().
 $MGMT_ADAPTER_NAME = "vEthernet (Ethernet*"
+$CNI_RELEASE_VERSION = 'v0.8.5'
```

Comment:
why is this needed?

Reply:
This was just moving a constant to the top of the file, but this is now obsolete after rebasing.